doc/source/tutorials/local/Adding_new_data/Adding_observation_data.rst

```diff
@@ -0,0 +1,306 @@
+Adding observation data
+=======================
+
+This tutorial demonstrates how OpenGHG can be used to process new
+measurement data, search the data present and to retrieve this for
+analysis and visualisation.
+
+0. Using the tutorial object store
+----------------------------------
+
+To avoid adding the example data we use in this tutorial to your normal
+object store, we need to tell OpenGHG to use a separate sandboxed object
+store that we'll call the tutorial store. To do this we use the
+``use_tutorial_store`` function from ``openghg.tutorial``. This sets the
+``OPENGHG_TUT_STORE`` environment variable for this session and won't
+affect your use of OpenGHG outside of this tutorial.
```

Comment:
Question about this functionality - does this still apply now the tutorial has been moved to RST and the user will be running this themselves outside of a notebook? What does "this session" mean in this new context?